x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/ml/datafeed_apm_metrics.json

```diff
@@ -0,0 +1,99 @@
+{
+  "job_id": "JOB_ID",
+  "indices": [
+    "INDEX_PATTERN_NAME"
+  ],
+  "chunking_config" : {
+    "mode" : "off"
+  },
+  "query": {
+    "bool": {
+      "filter": [
+        { "term": { "processor.event": "metric" } },
+        { "term": { "metricset.name": "transaction" } },
+        { "terms": { "_tier": [ "content", "hot", "warm" ]} }
```

Comment:
The `_tier` part doesn't need to be added to the datafeed query - the datafeed queries should be protected by the time range used to run the job.